SA3/qa_web/static/js/base.js

```diff
@@ -31,6 +31,40 @@ function vote_callback(event){
   });
 }
 
+function post_reply(event) {
+  var comment_button = event.currentTarget;
+  var form_id;
+  var content_id;
+  var modal_title;
+  var modal_content;
+  var reply_type = comment_button.id.slice(4); //4 since character deciding whether it's answer or comment is 5th character.
+
+  $("#wmd-input").val("");                  //Clear textbox each time we open the modal (since closing the modal beforehand presumes that user didn't want to reply)
+  $("#wmd-preview").html("");               //Similar story with preview pane.
+  if (reply_type[0] == "A") {
+    form_id = "answer_form";
+    $("#modalTitle").html("Posting Answer");
+    $("#postTitle").html("");
```

Comment:
Why not display the question here?

Reply:
I figured that the end user would be confused on whether they are answering or commenting on a question but I suppose the header of the modal will prevent such a situation

This is something I can modify pretty fast so I'll get to that in a bit.